lib/build.gradle

```diff
@@ -89,6 +89,7 @@ dependencies {
 	compatDiktat2Dot0Dot0CompileOnly "com.saveourtool.diktat:diktat-runner:2.0.0"
 	// flexmark
 	flexmarkCompileOnly 'com.vladsch.flexmark:flexmark-all:0.64.8'
+	implementation 'com.vladsch.flexmark:flexmark-util-data:0.64.8'
```

Comment:
Has to be `flexmarkCompileOnly`, not `implementation`. Spotless has *way* too many dependencies to deal with all the transitive conflicts. You can only use flexmark classes inside of `FlexmarkFormatterFunc`, which gets called with a special classloader after Gradle/Maven has done the dependency resolution.

Reply:
@nedtwigg okay, got it

Reply:
@nedtwigg I'm a bit confused. If we 're using only `flexmarkCompileOnly` it means that I can't get access to `flexmark-util-data` API inside of [FlexmarkStep](https://github.com/diffplug/spotless/blob/main/lib/src/main/java/com/diffplug/spotless/markdown/FlexmarkStep.java), which makes it impossible to run with custom configuration or am I wrong?